src/input_common/settings.h

```diff
@@ -339,6 +339,8 @@ enum class ControllerType {
     DualJoyconDetached,
     LeftJoycon,
     RightJoycon,
+    GameCube,
+    Pokeball,
     Handheld,
```

Comment:
```suggestion
    LeftJoycon,
    RightJoycon,
    Handheld,
    GameCube,
    Pokeball,
```

These should be placed below handheld to preserve compatibility